.cirrus.yml

```diff
@@ -69,11 +75,65 @@ low_scale_task:
   upload_caches:
     - runtime
 
+low_scale_task:
+  depends_on:
+    - runtime_builder
+
+  << : *COMMON_TASK_TEMPLATE
+
+  compute_engine_instance:
+    matrix:
+      - image_project: fedora-cloud
+        image: family/fedora-cloud-38
+      - image_project: ubuntu-os-cloud
+        image: family/ubuntu-2304-amd64
+    platform: linux
+    memory: 8G
+    disk: 40
```

Comment:
Looking at the resource usage, we could, likely reduce the cpu count from the default 2 to 1.
We probably don't need 8G of RAM as well, but I'm not sure why ubuntu builds eat almost 4, while fedora builds only use ~500 MB.